Control coverage/Feature/Storage.md

```diff
@@ -223,13 +223,13 @@ TLS provides privacy and data integrity between client and server. Using approve
 > Any of the following conditions is met.
 > * Minimum TLS version is not set (default 1.0).
```

Comment:
better to write if min TLS version is less than 1.2 (configured min required tls version)